### ISSUE TITLE
CuBLAS backend

### ISSUE THREAD

**Issue by GallagherCommaJack** [mstksg/tensor-ops#6]
Given that bindings [exist](https://hackage.haskell.org/package/cublas-0.2.1.0/docs/Foreign-CUDA-Cublas.html), how much work do you think it would take to make a BLAS instance?

**Comment by mstksg** [comment]
At this point it doesn't look too tricky; all it'd take is a BLAS instance.  The only thing is the fact that "monadic" backends aren't supported yet; everything might have to routed through unsafePerformIO :|

**Comment by GallagherCommaJack** [comment]
Yeah, that's less than ideal. That in mind, [accelerate](https://hackage.haskell.org/package/accelerate) might be a better way to get it onto GPU.
